Source/gg2/Scripts/Events/doEventPlayerDeath.gml

```diff
@@ -239,7 +239,15 @@ with(victim.object) {
     }
 }
 
-if (global.gg_birthday){
+if (has_crown) {

```

Comment:
This is actually buggy and crashes a client as soon as someone dies, since the victim object is not always the context for this script.